client/app/components/dialog-editor-modal-field-template/tag-control.html

```diff
@@ -0,0 +1,71 @@
+<!-- tag-contorl -->
+<div pf-form-group pf-label="{{'Required'|translate}}">
+  <input bs-switch
+         ng-model="vm.modalData.required"
+         type="checkbox"
+         switch-on-text="{{'Yes'|translate}}"
+         switch-off-text="{{'No'|translate}}">
+</div>
+<div pf-form-group pf-label="{{'Read only'|translate}}">
+  <input bs-switch
+         ng-model="vm.modalData.read_only"
+         type="checkbox"
+         switch-on-text="{{'Yes'|translate}}"
+         switch-off-text="{{'No'|translate}}">
+</div>
+<div pf-form-group pf-label="{{'Visible'|translate}}">
+  <input bs-switch
+         ng-model="vm.modalData.visible"
+         type="checkbox"
+         switch-on-text="{{'Yes'|translate}}"
+         switch-off-text="{{'No'|translate}}">
+</div>
+<div pf-form-group pf-label="{{'Auto refresh on change'|translate}}">
+  <input bs-switch
+         ng-model="vm.modalData.trigger_auto_refresh"
+         type="checkbox"
+         switch-on-text="{{'Yes'|translate}}"
+         switch-off-text="{{'No'|translate}}">
+</div>
+<div pf-form-group pf-label="{{'Category'|translate}}">
+  <select class="form-control" ng-model="vm.modalData.options.category_id"
+          ng-options="category.id.toString() as category.description for category in vm.categories.resources">
+    <option selected="selected" value="" translate>None</option>
+  </select>
+</div>
+<div pf-form-group pf-label="{{'Single value'|translate}}">
+  <input bs-switch
+         ng-model="vm.modalData.options.force_single_value"
+         type="checkbox"
+         switch-on-text="{{'Yes'|translate}}"
+         switch-off-text="{{'No'|translate}}">
+</div>
+<div pf-form-group pf-label="{{'Value type'|translate}}">
+  <select class="form-control" ng-model="vm.modalData.data_type">
+    <option value="integer" translate>Integer</option>
+    <option selected="selected" value="string" translate>String</option>
+  </select>
+</div>
+<div pf-form-group pf-label="{{'Sort by'|translate}}">
+  <select class="form-control" ng-model="vm.modalData.options.sort_by">
+    <option value="none" translate>None</option>
+    <option value="description" translate>Description</option>
+    <option value="value" translate>Value</option>
+  </select>
+</div>
+<div pf-form-group pf-label="{{'Sort order'|translate}}">
+  <select class="form-control" ng-model="vm.modalData.options.sort_order">
+    <option selected="selected" value="ascending" translate>Ascending</option>
+    <option value="descending" translate>Descending</option>
+  </select>
+</div>
+<!-- commented, as there is no way to load the data from API
```

Comment:
Hi @romanblanco, if you add the `attributes=children` to the call, you should be able to retrieve those values.